purchase_request/demo/purchase_request_demo.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2016 Eficent Business and IT Consulting Services S.L.
+     License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl-3.0) -->
+<openerp>
+    <data noupdate="0">
```

Comment:
Most of the time when using noupadate= into an odoo demo file, that's to specify it is noupdate="1" as https://github.com/odoo/odoo/blob/8.0/addons/account/demo/account_minimal.xml#L3